install/install.py

```diff
@@ -228,14 +252,32 @@ def danger(cls, string):
 
 
 if __name__ == "__main__":
+    root_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+    pid = os.getpid()
+
+    # custom scripts
+    scripts_path = os.path.join(root_dir, "scripts")
+
+    # OWTF python libraries
+    owtf_pip = os.path.join(root_dir, "install", "owtf.pip")
+
+    # Restricted tools and dictionaries which are distro independent
+    restricted_cfg = os.path.join(root_dir, "install", "distro-independent.cfg")
+
+    # Various distros and install scripts
+    distros_cfg = os.path.join(root_dir, "install", "linux-distributions.cfg")
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--no-user-input', help='run script with default options for user input', action="store_true")
+    parser.add_argument('--core-only', help='install only owtf dependencies, skip optional tools', action="store_true")
+
     Colorizer.normal("[*] Great that you are installing OWTF :D")
     Colorizer.warning("[!] There will be lot of output, please be patient")
 
-    RootDir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
-    installer = Installer(RootDir)
-    Colorizer.info("[DEBUG] Last commit hash: %s" % installer.version(RootDir))
-    installer.check_sudo()
-    installer.install(sys.argv[1:])
+    Colorizer.info("[DEBUG] Last commit hash: %s" % zest_version())
+    check_sudo()
+    install(sys.argv[1:])
 
     Colorizer.success("[*] Finished!")
     Colorizer.info("[*] Start OWTF by running './owtf.py' in parent directory")
+
```

Comment:
- [x] One newline is required according to PEP8 - there were two, removed one, retained one.